src/jit/codegencommon.cpp

```diff
@@ -87,9 +87,9 @@ CodeGenInterface *getCodeGenerator(Compiler *comp)
 
 // CodeGen constructor
 CodeGenInterface::CodeGenInterface(Compiler* theCompiler) :
+    regSet(theCompiler, gcInfo),
     compiler(theCompiler),
-    gcInfo(theCompiler),
-    regSet(theCompiler, gcInfo)
+    gcInfo(theCompiler)
```

Comment:
I think we should instead reorder the definitions in the class so that gcInfo is before regSet. (Notice that regSet takes the gcInfo as an argument). Not sure if there is a real problem but seems like regSet should be able to assume it has an initialized gcInfo.